finetune/base.py

```diff
@@ -247,12 +248,12 @@ def _predict(self, *Xs, max_length=None):
             max_length = max_length or self.config.max_length
             for xmb, mmb in self._infer_prep(*Xs, max_length=max_length):
                 output = self._eval(self.predict_op,
-                    feed_dict={
-                        self.X: xmb,
-                        self.M: mmb,
-                        self.do_dropout: DROPOUT_OFF
-                    }
-                )
+                                    feed_dict={
+                                        self.X: xmb,
+                                        self.M: mmb,
+                                        self.do_dropout: DROPOUT_OFF
```

Comment:
is this some autoformatter that just got run on all of our files?

Reply:
i don't know why it bugs me but generally prefer the orig. style.  instead of what I call the hammer of doom (indenting all arguments on additional lines to level of indentation of function call above).

Reply:
```
                output = self._eval(
                    self.predict_proba_op,
                    feed_dict={
                        self.X: xmb,
                        self.M: mmb,
                        self.do_dropout: DROPOUT_OFF
                    }
                )
```